perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_facility_attribute_def_def_m365AllowedLicensesPrioritiesTest.java

```diff
@@ -0,0 +1,214 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.Facility;
+import cz.metacentrum.perun.core.api.Resource;
+import cz.metacentrum.perun.core.api.exceptions.AttributeNotExistsException;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+import static org.mockito.Mockito.*;
+
+public class urn_perun_facility_attribute_def_def_m365AllowedLicensesPrioritiesTest {
+	private static urn_perun_facility_attribute_def_def_m365AllowedLicensesPriorities classInstance;
+	private static PerunSessionImpl session;
+	private static Facility facility;
+	private static Attribute attributeToCheck;
+
+	@Before
+	public void setUp() {
+		classInstance = new urn_perun_facility_attribute_def_def_m365AllowedLicensesPriorities();
+		session = mock(PerunSessionImpl.class, RETURNS_DEEP_STUBS);
+		facility = new Facility();
+		attributeToCheck = new Attribute();
+		attributeToCheck.setFriendlyName("m365AllowedLicensesPriorities:test");
```

Comment:
Not sure why the `test` namespace is here, the attribute does not use namespaces. It should not do any harm, but no reason to have it here (same for the other test classes)